Session2.Rmd

```diff
@@ -148,30 +153,26 @@ In the case of `point_Delft` those columns are `"osm_id"`, `"highway"` and `"geo
 names(lines_Delft)
 ```
 
-We can also preview the content of the object with the `head()` function.
+We can also preview the content of the object with the `head()` function, which is in the case of an `sf` object the same as examining the object directly.
 
 ```{r}
 head(lines_Delft)
 ```
 
-### Challenge
+### Challenge 2 (3 minutes)
 
-Explore the attributes associated with the `point_Delft` and `boundary_Delft` spatial objects.
+Explore the attributes associated with the `point_Delft` spatial object.
 
-1. How many attributes does each have?
+1. How many attributes does it have?
 2. What types of leisure points do the points represent? Give three examples.
-3. Which of the following is NOT an attribute of the point_Delft data object?
+3. Which of the following is NOT an attribute of the `point_Delft` data object?
 
   A) location B) leisure C) osm_id
 
 ```{r}
 ncol(point_Delft)
 ```
 
-```{r}
-ncol(boundary_Delft)
-```
-
 Using the `$` operator already introduced in the morning, we can examine the content of a single variable. We only see two values and `NA`s with the `head()` function, so we can [either] increase the number of rows we want to see[, use the function `na.omit()` to remove `NA`s completely or use the `unique()` function to only see the first occurrence of each distinct value (note that NA will still be included once). This is an example of how in R you can do things in many different ways.] Printing the object will also give us the first 10 rows.
```

Comment:
...We only see two values and `NA`s with the `head()` function, so we can [either] increase the number of rows we want to see**[,** ...
Is this an extra bracket?

Reply:
That was an optional sentence segment. I removed the square brackets.